### PR TITLE
NFT module

### PR DESCRIPTION
**Pull request by marbar3778**
- Added nft module
- add license
- add testing makefile
- add go.mod

Signed-off-by: Marko Baricevic <marbar3778@yahoo.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,19 @@
...
-To find out more about the Cosmos-SDK, you can find documentation [here](https://cosmos.network/docs/).
+[![license](https://img.shields.io/github/license/cosmos/cosmos-sdk.svg)](https://github.com/cosmos/modules/blob/master/LICENSE)
+
+**Note**: This repository is meant to house modules that are created outside of the [Cosmos-SDK](https://github.com/cosmos/cosmos-sdk) repository.
+
+**Note**: Requires [Go 1.13+](https://golang.org/dl/)
+
+## Quick Start
+
+To learn how the SDK works from a high-level perspective, go to the [SDK Intro](./docs/intro/intro.md).
```
Comment (fedekunze) [inline README.md]:
```suggestion
To learn how the SDK works from a high-level perspective, go to the [SDK Intro](https://github.com/cosmos/cosmos-sdk/docs/intro/intro.md).
```

File: README.md [hunk 2]
```diff
@@ -1,5 +1,19 @@
...
+
+If you want to get started quickly and learn how to build on top of the SDK, please follow the [SDK Application Tutorial](https://github.com/cosmos/sdk-application-tutorial). You can also fork the tutorial's repo to get started building your own Cosmos SDK application.
+
+For more, please go to the [Cosmos SDK Docs](./docs/README.md)
```
Comment (fedekunze) [inline README.md]:
```suggestion
For more, please go to the [Cosmos SDK Docs](https://github.com/cosmos/cosmos-sdk/docs/README.md)
```

File: stable/nft/app/app.go [hunk 3]
```diff
@@ -0,0 +1,301 @@
+package simapp
```
Comment (fedekunze) [inline stable/nft/app/app.go]: uhhhh why do we have an app defined on the module ? I don't see the need to define one for each module.

Also, the NFT module is not `stable`. It still has performance issues that need to be resolved so I'd move it to `beta`

File: README.md [hunk 4]
```diff
@@ -1,7 +1,22 @@
...
-This repo is meant to house modules that are created outside of the [Cosmos-SDK](https://github.com/cosmos/cosmos-sdk).
+![banner](cosmos-sdk-image.jpg)
 
+[![license](https://img.shields.io/github/license/cosmos/cosmos-sdk.svg)](https://github.com/cosmos/modules/blob/master/LICENSE)
+
+**Note**: This repository is meant to house modules that are created outside of the [Cosmos-SDK](https://github.com/cosmos/cosmos-sdk) repository.
+
+**Note**: Requires [Go 1.13+](https://golang.org/dl/)
+
+## Quick Start
```
[the remark this answers is not in the document]
Comment (marbar3778) [reply README.md]: renamed to beta, will see what others say with this change

[the remark this answers is not in the document]
Comment (marbar3778) [reply README.md]: seems like people want to stick to incubator.

File: stable/nft/app/app.go (hunk 3 above)
Comment (marbar3778) [reply stable/nft/app/app.go]: lets have this discussion in the chat, but not block this PR

Comment (marbar3778) [reply stable/nft/app/app.go]: https://github.com/cosmos/modules/issues/8#issue-505171877

**Pull request merged**